### PR TITLE
Document SHOW SEQUENCES.

### PR DESCRIPTION
**Pull request by knz**
To go along with https://github.com/cockroachdb/cockroach/pull/35215.

Would welcome suggestion as to how to generate the SVG diagram (the bnf file is created by the other PR).

### CODE REVIEW

File: v19.1/show-sequences.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/show_sequences.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------|------------
+`name` | The name of the database for which to list sequences. When omitted, the schemas in the [current database](sql-name-resolution.html#current-database) are listed.
```
Comment (rmloveland) [inline v19.1/show-sequences.md]: Did you mean

> when omitted, the _sequences_ in the current database ...

?

**Comment by rmloveland** [comment]
> Would welcome suggestion as to how to generate the SVG diagram (the bnf file is created by the other PR).

The instructions for doing so are on our GH wiki at [SQL grammar railroad diagram changes](https://github.com/cockroachdb/docs/wiki/SQL-Grammar-Railroad-Diagram-Changes).

Long ago I wrote a script to do this automatically though - so I'm happy to do that part for you.

And thanks for the PR!

**Comment by rmloveland** [comment]
For reference here are the commands I ran that:
- build the `docgen` tool
- generate the SVG diagrams
- dump the diagrams in the (in this case) 19.1 diagrams folder of the docs repo

```
$ cd /Users/rloveland/go/src/github.com/cockroachdb/cockroach
$ go install ./pkg/cmd/docgen
$ cd /Users/rloveland/go/src/github.com/cockroachdb/cockroach
$ make bin/.docgen_bnfs
Running make with -j4
GOPATH set to /Users/rloveland/go
$ cd /Users/rloveland/go/src/github.com/cockroachdb
$ /Users/rloveland/go/bin/docgen grammar svg /Users/rloveland/go/src/github.com/cockroachdb/cockroach/docs/generated/sql/bnf/ /Users/rloveland/work/code/docs/_includes/v19.1/sql/diagrams/ --filter=sequences

generating svg of show_sequences (/Users/rloveland/go/src/github.com/cockroachdb/cockroach/docs/generated/sql/bnf/show_sequences.bnf)
```

File: v19.1/show-sequences.md (hunk 1 above)
Comment (knz) [reply v19.1/show-sequences.md]: yes sorry about that.

**Comment by rmloveland** [comment]
I'll take this one over from Raphael (discussed offline).

**Comment by rmloveland** [comment]
@jseldess this should be good to go; can use a look whenever you have a moment.

Direct link: [SHOW SEQUENCES](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/a66bd2873fec1a1e6e65111369a59886c00188cb/dev/show-sequences.html)

**Pull request merged**